routers/user/kitspace_auth.go

```diff
@@ -70,12 +71,31 @@ func KitspaceSignUp(ctx *context.Context, form auth.RegisterForm) {
 		return
 	} else {
 		log.Trace("Account created: %s", u.Name)
+	}
+
+	// Send confirmation email
+	if setting.Service.RegisterEmailConfirm && u.ID > 1 {
+		mailer.SendActivateAccountMail(ctx.Locale, u)
+
+		response := make(map[string]string)
+		response["email"] = u.Email
+		response["ActiveCodeLives"] = timeutil.MinutesToFriendly(
+			setting.Service.ActiveCodeLives,
+			ctx.Locale.Language(),
+		)
+
+		if err := ctx.Cache.Put("MailResendLimit_"+u.LowerName, u.LowerName, 180); err != nil {
+			log.Error("Set cache(MailResendLimit) fail: %v", err)
+		}
+
+		ctx.JSON(http.StatusOK, response)
+	} else {
 		response := make(map[string]bool)
```

Comment:
This else block for local development, I'll remove it once I integrate the frontend forms.

Reply:
What's the status of this?